tool/site/client-react/src/components/prescriptionActions.tsx

```diff
@@ -40,6 +42,11 @@ const PrescriptionActions: React.FC<PrescriptionActionsProps> = ({
           Return prescription
         </ActionLink>
       )}
+      {withdraw && (
+        <ActionLink href={`${baseUrl}dispense/withdraw?prescription_id=${prescriptionId}`}>
+          Withdraw prescription
+        </ActionLink>
+      )}
       {dispense && (
```

Comment:
Dispense should probably be above withdraw here, to be consistent with release. Return is undoing a release so it's after release in the list. Withdraw is undoing 1 or more dispenses so would make sense to be after "Dispense"

Reply:
Moved it around a little